legacy/definition_test.go

```diff
@@ -172,8 +170,8 @@ func TestActionMigration(t *testing.T) {
 
 		migratedAction := migratedFlow.Nodes()[0].Actions()[0]
 		migratedActionEnvelope, _ := utils.EnvelopeFromTyped(migratedAction)
-		migratedActionJSON, _ := utils.JSONMarshalPretty(migratedActionEnvelope)
-		expectedActionJSON, _ := utils.JSONMarshalPretty(test.ExpectedAction)
+		migratedActionJSON, _ := utils.JSONMarshal(migratedActionEnvelope)
+		expectedActionJSON, _ := utils.JSONMarshal(test.ExpectedAction)
```

Comment:
the assert library prints these on a single line so better not to have all the escaped `\n\t` chars